test/amqp-integration-test.py

```diff
@@ -94,9 +94,11 @@ def verify_ocsp_good(certFile, url):
 def verify_ocsp_revoked(certFile, url):
     output = get_ocsp(certFile, url)
     if not re.search(": revoked", output):
-        print "Expected OCSP response 'revoked', got something else."
-        die(ExitStatus.OCSPFailure)
-    pass
+        if not re.search(": good", output):
```

Comment:
Please add a comment explaining what's going on here. I get it, but it took me a minute.